rh-tictactoe/src/redux/login/reducer.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+import { actions } from './actions';
+
+const initialState = {
+  email: '',
+  token: ''
+};
```

Comment:
you never, i repeat, NEVER, save the token on the store.
Did i remember to say never? Well yeah, never